docs/get-started.md

```diff
@@ -4,14 +4,43 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
+Welcome to Temporal's user documentation!
+
 Temporal reimagines state-dependent service-orchestrated application development, and so much more.
 
 ## Get started
 
-This is page will be dedicated to directing users on how to get started with Temporal.
+Ready to develop? [Install the Temporal server](install-temporal-server) and choose your SDK.
+
+→ [Go SDK](go-quick-start)
 
-This is placeholder content.
+→ [Java SDK](java-quick-start)
 
 ## Learn about Temporal
 
-etc...
+Building applications with Temporal is a paradigm shift. If you are new to Temporal, or workflow applications in general, get caught up on the [core concepts](concepts-overview) and common [use cases](use-cases-overview).
```

Comment:
The first sentence:

"Building applications with Temporal is a paradigm shift. "

should definitely be removed, we want to avoid communicating things in a negative way. Obviously what you said is 100% true. If you want to communicate the same message with a positive spin consider something like:

"Temporal makes it a breeze to write reliable applications. Problems that developers are traditionally expected to solve themselves are handled implicitly by the system. While Temporal brings some huge advantages to developing applications, for some it requires a little getting used to..."

Or something

Reply:
Also maybe split out the use case part. into a sentence.

"Wondering if your use case is a good fit for Temporal?" And then link use cases and maybe a forum link if the use case isn't covered in our docs

Reply:
> The first sentence:
> 
> "Building applications with Temporal is a paradigm shift. "
> 
> should definitely be removed, we want to avoid communicating things in a negative way. Obviously what you said is 100% true. If you want to communicate the same message with a positive spin consider something like:
> 
> "Temporal makes it a breeze to write reliable applications. Problems that developers are traditionally expected to solve themselves are handled implicitly by the system. While Temporal brings some huge advantages to developing applications, for some it requires a little getting used to..."
> 
> Or something

I disagree. There is nothing negative about that statement. It is meant to be very to the point - otherwise we are creating confusing messaging within our docs - If we are talking about our homepage, then yes, I would agree with you. But in our docs, I think this type of clarity is very important. I think this strong, clear message helps to set expectations for our users when they are deciding which docs to read first.

Reply:
> Also maybe split out the use case part. into a sentence.
> 
> "Wondering if your use case is a good fit for Temporal?" And then link use cases and maybe a forum link if the use case isn't covered in our docs

I agree with this -